tests/oss-fuzz/build.sh

```diff
@@ -73,10 +74,10 @@ mkdir build
 cd build
 if [ "$FUZZING_ENGINE" == "libfuzzer" ]
 then
-  export CXXFLAGS="${CXXFLAGS} -DFUZZTEST_COMPATIBILITY_MODE"
-  export EXTRA_CMAKE_FLAGS="-DAVIF_ENABLE_FUZZTEST=ON -DFUZZTEST_COMPATIBILITY_MODE=libfuzzer"
+  CXXFLAGS="${CXXFLAGS} -DFUZZTEST_COMPATIBILITY_MODE"
+  EXTRA_CMAKE_FLAGS="-DAVIF_ENABLE_FUZZTEST=ON -DFUZZTEST_COMPATIBILITY_MODE=libfuzzer"
```

Comment:
Optional: We can also initialize `EXTRA_CMAKE_FLAGS` to an empty string before the if statement, and append to it here. This will allow us to get rid of the else branch.

Reply:
Done